src/scores/utils.py

```diff
@@ -35,9 +35,22 @@ class DimensionError(Exception):
     Dataset objects that do not have compatible dimensions.
     """
 
+
 def gather_dimensions(fcst_dims, obs_dims, weights_dims=None, reduce_dims=None, preserve_dims=None):
     """
     Establish which dimensions to reduce when calculating errors but before taking means
+
+    Args:
+        fcst_dims (Iterable[str]): Forecast dimensions inputs
+        obs_dims (Iterable[str]): Observation dimensions inputs.
+        weights_dims (Iterable[str]): Weight dimension inputs.
+        reduce_dims (Iterable[str]): Dimensions to reduce.
```

Comment:
reduce and preserve can both either be a vanilla string (e.g. 'all') or an iterable of strings.

Reply:
Fixed in 52410f0